api_docs/deploying-bots.md

```diff
@@ -44,22 +44,21 @@ Botserver interaction are:
 
 1. The Zulip server sends a POST request to the Botserver on `https://bot-server.example.com/`:
 
-    ```json
+    ```
```

Comment:
What's the thinking on removing the `json` language formatting? I can imagine a few reasons, but am not sure what the motivation was.

Reply:
> What's the thinking on removing the `json` language formatting? I can imagine a few reasons, but am not sure what the motivation was.

Code highlighting was broken as noted here https://github.com/zulip/zulip/pull/28895#discussion_r1498550383.

> I'm also a bit confused by the whitespace changes just below.

Good catch. I think I unintentionally changed the whitespace formatting while trying to get the syntax highlighting to work. I reverted the changes.

Reply:
OK -- can you start a thread in #issues about syntax highlighting being broken here? I doubt it's a hard fix, given that I think it does work in other places.